sycl/include/sycl/builtins.hpp

```diff
@@ -1084,10 +1126,8 @@ detail::enable_if_t<detail::is_ugeninteger<T>::value, T> abs(T x) __NOEXC {
 
 // ugeninteger abs (geninteger x)
 template <typename T>
-detail::enable_if_t<detail::is_igeninteger<T>::value,
-                    detail::make_unsigned_t<T>>
-abs(T x) __NOEXC {
-  return __sycl_std::__invoke_s_abs<detail::make_unsigned_t<T>>(x);
+detail::enable_if_t<detail::is_igeninteger<T>::value, T> abs(T x) __NOEXC {
```

Comment:
Why do we change return type here? If it's valid change, then the comment on line 1127 needs to be updated.

Reply:
This is according to SYCL 2020. Updated line 1127